CHANGELOG.rst

```diff
@@ -30,7 +30,7 @@ Added
   directtory in which ``egon-data`` is started, so it's probably best to
   run ``egon-data`` in a dedicated directory.
   There's also the new function `egon.data.config.settings` which
-  returns the current configuration settings.
+  returns the current configuration settings. See :pr:`159`
```

Comment:
Sorry to curb your enthusiasm here, especially since I'm the one introducing the `:issue:` and `:pr:` roles, but unfortunately these roles are only rendered by Sphinx, so it's probably better to not use them in reST files which will be viewed both, on GitHub and on Read the Docs.
So manually expanded `:pr:`s in "CHANGELOG.rst" with 5b4b3734e78f0c8353562fccce977b6861fb46a4 .